src/backend-api/Controllers/LoginController.cs

```diff
@@ -0,0 +1,201 @@
+﻿using System;
+using System.Collections.Generic;
+using System.DirectoryServices.AccountManagement;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.IdentityModel.Tokens.Jwt;
+using System.Text;
+using Microsoft.IdentityModel.Tokens;
+using System.Configuration;
+using Microsoft.IdentityModel.Logging;
+using backend_api.Helpers;
+using Microsoft.Extensions.Options;
+using backend_api.Models;
+using backend_api;
+
+namespace EFGetStarted.AspNetCore.ExistingDb.Controllers
+{
+
+    [Route("api/Login")]
+    [ApiController]
+    public class LoginController : ControllerBase
+    {
+        // Instance of appsettings so that we can pull in our secret to create tokens
+        private readonly AppSettings _appSettings;
+        // Instance of the Database so we can read it and  update it
+        private readonly ITInventoryDBContext _context;
+
+        //Constructor for appsettings and database
+        public LoginController(IOptions<AppSettings> appSettings, ITInventoryDBContext context)
+        {
+            _context = context;
+            _appSettings = appSettings.Value;
+        }
+
+        /* Post: api/Login
+        * Returns {
+        *          Refresh Token,
```

Comment:
Can you say what types there are for the things return? For example FirstName: string, IsAdmin: bool.